include/swift/AST/SearchPathOptions.h

```diff
@@ -505,11 +496,11 @@ class SearchPathOptions {
   /// Path to in-process plugin server shared library.
   std::string InProcessPluginServerPath;
 
-  /// Don't look in for compiler-provided modules.
-  bool SkipRuntimeLibraryImportPaths = false;
+  /// Don't automatically add any import paths.
+  bool SkipAllImportPaths = false;
```

Comment:
Nit: Maybe something like `SkipAllImplicitImportPaths`. Otherwise it reads weird to me to have a flag in here to skip *all* import paths.

Reply:
I like it 👍